### PR TITLE
IDVA5-1484 remove hard coded verification documents

### PR DESCRIPTION
**Pull request by ttingle-ch**
- Updated the verification documents to match the values expected by the verification api
- Updated the payload send to the verification api on confirming identity to match the new verification evidence structure

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/services/formatService.ts
Comment (rdas-ch) [inline src/services/formatService.ts:1]: photoimmigrationDoc, needs camel-casing ?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/services/identityVerificationService.ts [hunk 1]
```diff
@@ -76,12 +76,17 @@ export class IdentityVerificationService {
...
 
+        const documentsChecked = clientData.documentsChecked!;
+        const verificationEvidence = documentsChecked.map((document) => {
+            return { type: document };
+        });
+
         return {
-            // below 3 fields are hardcoded. Need to replace with actual logic in future
+            // below 2 fields are hardcoded. Need to replace with actual logic in future
             acspId: "1234567890",
```
Comment (swalke-ch) [inline src/services/identityVerificationService.ts:86]: I know it's not part of this ticket, but can we assign acspId here if its part of session value now ?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/services/identityVerificationService.ts (hunk 1 above)
Comment (ttingle-ch) [reply src/services/identityVerificationService.ts]: Updated

File: src/services/formatService.ts
Comment (ttingle-ch) [reply src/services/formatService.ts:1]: done

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by swalke-ch — APPROVED** [review]

**Review by cbroadley-ch — APPROVED** [review]

**Pull request merged**